api-reference/50 Common/Object Structures/FileSystemItem/FileSystemItem.md

```diff
@@ -6,7 +6,6 @@ type: Object
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+An object that provides information about a file system item (file or folder) in the [FileManager](/api-reference/10%20UI%20Widgets/dxFileManager '/Documentation/ApiReference/UI_Widgets/dxFileManager/') widget.
```

Comment:
I believe in every place except here "_a_ file system item" should be "_the_ file system item", because we introduce it in this sentence and refer to the same file system item in the rest of the descriptions.